storage/preferences/src/main/java/de/danoeh/antennapod/storage/preferences/UserPreferences.java

```diff
@@ -926,6 +929,20 @@ public static void setQueueKeepSortedOrder(SortOrder sortOrder) {
                 .apply();
     }
 
+    public static DateSortType getInboxSortOrder() {
```

Comment:
The UserPreferences class is already extremely large. Additionally, it needs to be a dependency of nearly every module. Instead, I would prefer to store the inbox sort order directly in the inbox. Something similar is already done for the number of columns on the subscriptions screen, in case you want to have a look at how it can be done.